index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="nav-section">
+        <nav>
+          <ul class="main-nav">
+            <li>
+              <a href="#" class="text-white logo">My Logo</a>
+            </li>
+            <li class="setting-mobile">
+              <a href="#">
+                <img src="images/ham_setting.svg" alt="">
+              </a>
+            </li>
+          </ul>  
+        </nav>
+    </header>
+    <section class="bg-primary">
+        <div>
+            <p class="text-primary">
+              Hey There. I’m Ruby.
+            </p>
+            <p class="text-supportive">
+                I'm a Software Developer
+            </p>
```

Comment:
- [ ] **[OPTIONAL]** Great job making the changes and improvements to your project! I have a suggestion that could further improve the consistency and professionalism of your portfolio. Since your main title includes your name, it would be better to use an `h1` tag instead of a `p` tag inside this `div`. This helps to emphasize the importance of your name and make the structure of your portfolio more consistent. Keep up the good work! 💪